lib/logger/node.js

```diff
@@ -16,7 +16,7 @@ const Errors = require('../errors');
  */
 function Logger(options) {
   let winstonLogger;
-  const defaultFilePath = 'snowflake.log';
+  const defaultLogOutput = 'STDOUT';
```

Comment:
I don't know if we are allowed to do such a change. It would be OK if it was only in test scope.

Reply:
Reverted it and done the distinction of the transports in a slightly different manner.